lib/abstract_format.ml

```diff
@@ -941,6 +984,15 @@ and guard_test_assoc_of_sf sf : (guard_test_assoc_t, err_t) Result.t =
   | _ ->
      Err.create ~loc:[%here] (Err.Not_supported_absform ("guard_test_assoc", sf)) |> Result.fail
 
+and atom_or_wildcard_of_sf sf =
+  match sf with
+  | Sf.Tuple (3, [Sf.Atom "atom"; _; Sf.Atom field_name]) ->
```

Comment:
ditto.
Could you add `line` field into `AtomWildcardAtom`?

Reply:
yes